lib/msal-browser/src/utils/BrowserConstants.ts

```diff
@@ -50,3 +52,7 @@ export enum TemporaryCacheKeys {
     REQUEST_PARAMS = "request.params",
     SCOPES = "scopes"
 }
+
+export const DEFAULT_REQUEST: AuthorizationUrlRequest = {
+    scopes: [Constants.OPENID_SCOPE, Constants.PROFILE_SCOPE]
```

Comment:
If you pass empty array will these scopes get added automatically?

Reply:
We could have the logic downstream also use this same constant.

Reply:
I had the exact q. What is the conclusion?

Reply:
Feel free to resolve if addressed.

Reply:
@sameerag Yes they do get added downstream, PR was updated to have the downstream addition utilize this constant as well